.travis.yml

```diff
@@ -29,9 +29,8 @@ before_install:
 install:
   # Customise the testing environment
   # ---------------------------------
+  - conda install --channel ioos "Django>=1.7,<1.8" python-dateutil "matplotlib>=1.2.0" "netCDF4>=1.0.2" "rtree==0.7.0" jsonfield pytz djangorestframework pyugrid pyproj "pyaxiom>=0.0.9"
```

Comment:
I could not because there are requirements in the file that are not in conda.  I had the things not in conda installing before this using `pip` but they were installing different versions of Django and I didn't really feel like diving deep into dealing with using two different package management systems.